modules/ROOT/pages/configure-teams.adoc

```diff
@@ -0,0 +1,135 @@
+
+* Change the team name
+* Add or remove members from the team
+* Add or remove additional team maintainers
+* Move the team under different parent teams (if they maintain the other parent team)
+* Delete the team
+
+== Create a Child Team
+
+If you have the Organization Administrator permission in your root organization, you can create child teams under any existing parent teams, including the root team. Users who have the Organization Administrator permission in business groups can create child teams under the root team and then become team maintainers for that team. Additionally, team maintainers can create child teams under teams that they maintain.
+The child team inherits permissions from its parent team, but you configure additional permissions for its users.
+
+To create a child team in the Teams feature interface:
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+include::partial$include-nav-teams.adoc[]
+
+. Select *Create Team*.
+. Enter a unique and intuitive name for your child team. +
+You can use only alphanumeric characters, hyphens, commas, periods, and spaces. The character limit is 255.
++
+. Select a parent team.
+. Select *Save*.
+
+The child team appears under its parent team in the list of teams. You can also select a parent team and navigate to the *Child teams* tab to see a list of all child teams under that parent team.
+
+== Manage Team Permissions
+
+When you begin using the Teams feature, your organization’s main team is named “Everyone at <Your Organization>” and does not have any permissions assigned.
+If you have the Organization Administrator permission, you can configure permissions for this team in addition to child teams.
+If you invite users to “Everyone at <Your Organization>” without assigning any permissions to that team, your users cannot execute tasks or view assets in your organization.
+
+If you have the Organization Administrator permission in a specific business group but not your entire organization, you can assign these permissions to any of existing teams or users within your organization.
+
+To configure permissions for teams in the Teams feature interface:
+
+include::partial$include-nav-steps-team-maintainer.adoc[]
+
+. Click the team for which you want to configure permissions.
+. Click the *Permissions* tab.
+. In the *Permissions* section, click *Add permissions*
+. In the *Select Permissions* window, select the permissions that you want to add, and then select *Select contexts*.
+. Select the business groups and environments to which you want to apply the permissions, and then select *Review*. +
+The permissions you selected appear alongside the business groups and environments to which they apply.
++
+. Select *Apply permissions*. +
+The permissions you selected appear in the *Permissions* section for your team.
+
+== Manage Team Users
+
+If you have the Organization Administrator permission in your root organization, you can add members to existing teams using three methods:
```

Comment:
```suggestion
If you have the Organization Administrator permission in your root organization, you can add users to existing teams using three methods:
```
...since we are distinguishing between members and maintainers....